samples/javascript_nodejs/21.corebot-app-insights/bots/resources/welcomeCard.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "type": "AdaptiveCard",
+    "version": "1.0",
+    "body": [
+      {
+        "type": "Image",
+        "url": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQtB3AwMUeNoq4gUBGe6Ocj8kyh3bXa9ZbV7u1fVKQoyKFHdkqU",
+        "size": "stretch"
+      },
+      {
+        "type": "TextBlock",
+        "spacing": "medium",
+        "size": "default",
+        "weight": "bolder",
+        "text": "Welcome to Bot Framework!",
+        "wrap": true,
+        "maxLines": 0
+      },
+      {
+        "type": "TextBlock",
+        "size": "default",
+        "isSubtle": "yes",
```

Comment:
>yes [](start = 21, length = 3)

These should be "true" - checkout PR below..
[https://github.com/microsoft/BotBuilder-Samples/pull/1776](https://github.com/microsoft/BotBuilder-Samples/pull/1776)

Reply:
Feedback applied